services/pseudo_controller/src/main/resources/arion_data.json

```diff
@@ -0,0 +1,90 @@
+{
+  "ZGC_data": {
+    "name": "ZGC_test106",
+    "description": "ZGC_test106",
+    "ip_start": "192.168.20.100",
+    "ip_end": "192.168.20.115",
+    "port_ibo": "8300",
+    "overlay_type": "vxlan"
+  },
+   "NODE_data": [
+    {
+      "zgc_id": "zgc_id",
+      "description": "znode90",
+      "name": "znode90",
+      "ip_control": "192.168.20.190",
+      "id_control": "ubuntu",
+      "pwd_control": "Lab12345",
+      "inf_tenant": "ens4",
+      "mac_tenant": "52:54:00:5e:99:ac",
+      "inf_zgc": "ens3",
+      "mac_zgc": "52:54:00:f5:aa:52",
+      "arion_agent_location": "absolute_path_of_arion_agent_binary"
```

Comment:
Do we have a document to describe the meaning of each field? I remember Zeta has a doc which can be referenced here.

Reply:
hmm, I couldn't find the doc you are referring to. But The `NODE_data` refers to information for the wings.

Reply:
Could we provide a quick doc in a separate PR to detail the meaning of each field? Thanks. @zzxgzgz

Reply:
Sure, we can do that.